app/Http/Middleware/CheckIfTeacher.php

```diff
@@ -18,12 +19,15 @@ class ChekcIfTeacher
      */
```

Comment:
Je kan in de comments ook nog eventueel de `@param` en `@return` toevoegen. zo dat we snel en makkelijk weten wat de functie doet

```
/**
* Handle an incoming request.
*  @param $request Request dit is de request
* @param $next Closure dit is de closure
* @return data_type redirects de gebruiker
*/
```

Voor meer info: https://pear.php.net/package/PhpDocumentor/docs/latest/phpDocumentor/tutorial_tags.param.pkg.html